sdk/src/main/java/io/customer/sdk/android/CustomerIO.kt

```diff
@@ -1,11 +1,14 @@
-package io.customer.sdk
+package io.customer.sdk.android
```

Comment:
Moved old instance to temporary package to avoid breaking modules that haven't been refactored yet